velox/exec/OperatorTraceReader.cpp

```diff
@@ -32,10 +32,13 @@ OperatorTraceInputReader::OperatorTraceInputReader(
       pool_(pool),
       inputStream_(getInputStream()) {
   VELOX_CHECK_NOT_NULL(dataType_);
-  VELOX_CHECK_NOT_NULL(inputStream_);
 }
 
 bool OperatorTraceInputReader::read(RowVectorPtr& batch) const {
+  if (inputStream_ == nullptr) {
```

Comment:
Can we add a test for the null input stream?FilterProjectReplayerTest cannot prevent others to break the logic.